plugin/vite.config.template.ts

```diff
@@ -56,6 +58,16 @@ export const web =
     build: {
       outDir: `../../dist/web/${name}`,
     },
+    css: {
+      preprocessorOptions: {
+        less: {
+          javascriptEnabled: true,
+          modifyVars: {
+            "@primary-color": "#00BEBE",
+          },
+        },
+      },
+    },
```

Comment:
To keep styling simpler, please remove this and just change the color through the styled components.

Reply:
I will keep it as discussed

Reply:
https://github.com/eukarya-inc/reearth-plateauview/pull/29/commits/326aa3062b26c0e076d4cc56cf8ad8237c0cf1e2